.github/workflows/ci_verify_pr_contains_issue.yaml

```diff
@@ -0,0 +1,18 @@
+name: VerifyIssue
+
+on:
+  pull_request:
+    branches: [master]
```

Comment:
Any reason why you picked this too? It won't run anyways

Reply:
Oh I missed it. Thanks !